discourse-doctor

```diff
@@ -0,0 +1,374 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR
```

Comment:
I think `cd $DIR || exit` would be better

Reply:
Except then the script would quit without explanation. I guess that would happen only if they try to run the script without first changing to it and the directory where `discourse-doctor` lives did not have the X bit set for that user.

I'll stick bit after the root check, so it should never fail.

Reply:
It's up to you to write an explanation here before exiting. But I think it's a good practice to ensure cd was a success otherwise you could end up continuing the script thinking you are in a specific dir when you are not. And if today it makes sense, and nothing is risky after, who knows what will be put after in the future, that's why I recommended going with the safe choice here.